src/components/screen-view/FullScreenView.tsx

```diff
@@ -62,7 +64,10 @@ export function FullScreenView(props: Props) {
           backgroundColor={backgroundColor}
         />
       ) : (
-        <ChildrenInNormalScrollView {...props} />
+        <ChildrenInNormalScrollView
+          {...props}
+          backgroundColor={backgroundColor}
```

Comment:
I don't think it is intuitive that the background color of the children automatically fallbacks to the theme of the header. Also this is not the same behavoiur as in the `ChildrenWithParallaxScrollContent` where the background color of the children is just the default app background color (background_1). The parameter `backgroundColor` (could be named more specifically) in `ChildrenWithParallaxScrollContent` is to specify the color of the parallax content, and not the children.

I think the proper solution here could be to add a parameter like `childrenTheme` to the `FullScreenView`.

Reply:
Ok. And what TS-type do you suggest `childrenTheme` should be?

Reply:
In the header it is `color?: StaticColor;`. So `childrenColor` or `contentColor` typed to `StaticColor` may be appropriate here.